### PR TITLE
TD-1462/add-gpac

### PR DESCRIPTION
**Pull request by Tyk-ITS**
Adding terraform files for release branch management

### CODE REVIEW

**Review by alephnull — APPROVED** [review]

File: ci/repo-policy/main.tf [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  visibility                  = "public"
+  wiki                        = false
+  vulnerability_alerts        = true
+  squash_merge_commit_message = "COMMIT_MESSAGES"
+  squash_merge_commit_title   = "COMMIT_OR_PR_TITLE"
+  release_branches     = [
+{ branch    = "master",
+	reviewers = "2",
+	convos    = "false",
+	required_tests = ["1.15"]},
```
Comment (tbuchaillot) [inline ci/repo-policy/main.tf:38]: Can we add `CI tests` here?

File: ci/repo-policy/main.tf [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+{ branch    = "release-1.7",
+	reviewers = "2",
+	convos    = "false",
+	required_tests = ["1.15"]},
```
Comment (tbuchaillot) [inline ci/repo-policy/main.tf:42]: Can we add `CI tests` here?

**Review by tbuchaillot — CHANGES_REQUESTED** [review]

File: ci/repo-policy/main.tf (hunk 1 above)
Comment (ermirizio) [reply ci/repo-policy/main.tf:38]: you mean CI tests / Go 1.16 tests ??

**Comment by ermirizio** [comment]
@tbuchaillot I have done the changes can we approve and merge now?

**Review by tbuchaillot — APPROVED** [review]

**Pull request merged**